onnx_tf/backend.py

```diff
@@ -1085,7 +1085,7 @@ def handle_split(cls, node, input_dict):
     split = (tf.constant(node.attrs["split"]) if
              "split" in node.attrs else input_dict[node.inputs[1]])
     axis = node.attrs["axis"]
-    return [el for el in tf.split(input_dict[node.inputs[0]], split, axis)]
+    return [list(tf.split(input_dict[node.inputs[0]], split, axis))]
```

Comment:
I think we shouldn't need the outer brackets. It should simply be:
`list(tf.split(input_dict[node.inputs[0]], split, axis))`